docs/readthedocs/source/doc/UserGuide/develop.md

```diff
@@ -92,6 +92,24 @@ export PYTHONPATH=BigDL/python/dist/conf/spark-bigdl.conf:$PYTHONPATH
 The above environment variables should be available when running or debugging code in the IDE. When running applications in PyCharm, you can add runtime environment variables by clicking  __Run__ -> __Edit Configurations__; then in the __Run/Debug Configurations__ panel, you can add necessary environment variables to your applications.
 
 
+#### **1.3 Terminal Setup**
+
+Alternatively, for linux user you can set up environment variables mentioned above mantually, and we also provide a solution by following simple steps:
```

Comment:
Besides setting the environment variables mentioned above manually for Linux users, we also provide a solution to set them with a script.